packages/web/pages/blog/index.mdx

```diff
@@ -0,0 +1,11 @@
+import styles from "../../styles/Home.module.css";
+
+<>
+
+<main className={styles.main}>
```

Comment:
This style doesn't exist, so it just gets stripped out of the html. Probably best to just remove.